### PR TITLE
feat(native):Add openrtb native 1.2 support.

### PR DESCRIPTION
**Pull request by Fan-Xu**
The new package native implements a strict subset of OpenRTB Native 1.2 specification.
This package also omits extensive documentation over each type; 
for detailed explanation please
refer to  https://iabtechlab.com/standards/openrtb-native/
 https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf

### CODE REVIEW

File: openrtb/native/request/data.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package request
+
+import (
```
Comment (yingchen0706v) [inline openrtb/native/request/data.go:3]: singleline import

File: openrtb/native/request/request.go [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	// Scope:
+	//   optional
+	// Type:
+	//   integer
+	// Default:
+	//   1
+	// Description:
+	//   The number of identical placements in this Layout.
+	//   Refer Section 8.1 Multiplacement Bid Requests for further detail.
+	PlcmtCnt int64 `json:"plcmtcnt,omitempty"`
```
Comment (yingchen0706v) [inline openrtb/native/request/request.go:71]: The plcmt represents placement? Maybe we can use full name for variable name.

File: openrtb/native/request/request.go [hunk 3]
```diff
@@ -0,0 +1,156 @@
+package request
+
+import "github.com/Vungle/vungo/openrtb/native"
+
+// Request Object defines the native1 advertising opportunity available for bid via this bid request.
```
Comment (yingchen0706v) [inline openrtb/native/request/request.go:5]: is native1 typo or not?

**Review by yingchen0706v — APPROVED** [review]

**Review by xiaopeng0216 — APPROVED** [review]

**Comment by xiaopeng0216** [comment]
LGTM

**Pull request merged**